contracts/0.8.9/Accounting.sol

```diff
@@ -109,17 +108,13 @@ contract Accounting {
 
     /// @notice calculates all the state changes that is required to apply the report
     /// @param _report report values
-    /// @param _withdrawalShareRate maximum share rate used for withdrawal finalization
-    ///                             if _withdrawalShareRate == 0, no withdrawals are
-    ///                             simulated
     function simulateOracleReport(
```

Comment:
let's describe the oracle flow briefly here:
1. simulate with zero withdrawals
2. take returned values
3. calculate withdrawals
4. pass a real report

or smth